integration-tests/tests/pool/5.sTokenTransfer.spec.ts

```diff
@@ -0,0 +1,268 @@
+import { SorobanClient } from "../soroban.client";
+import {
```

Comment:
`debtTokenBalanceOf`, `debtTokenTotalSupply`, `setPrice`,  `withdraw` are not being used.